server/views/pages/adjustments/remand/view.njk

```diff
@@ -23,6 +23,11 @@
       <h1 class="govuk-heading-xl"><span class="govuk-caption-xl">Adjustments</span>
         Remand overview
       </h1>
+      {% if model.unusedDeductionMessage.unusedDeductionMessage == 'UNSUPPORTED' or  model.unusedDeductionMessage.unusedDeductionMessage == 'NOMIS_ADJUSTMENT' %}
```

Comment:
I've made it so the message on appears for MUD and RUD. There are only tickets for MUD and RUD. But I'll ask Alex about it for when he's back off holiday.